e4s_cl/cf/assets.py

```diff
@@ -96,8 +95,8 @@ def check_builtin_profile(system, configuration):
         raise ValueError(f"Profile {system} data is not a dictionary!"
                 " Profile import cancelled.")
 
-    attr = attributes()
-    for key in configuration.keys():
+    attr = profile.attributes()
```

Comment:
This is exactly what was wrong before. Use the `attributes` property from the `Profile` class in `e4s_cl.model.profile` instead of the `attributes` function in `e4s_cl.model.profile`.